start-build-env.sh

```diff
@@ -37,9 +37,17 @@ USER_ID=$(id -u "${USER_NAME}")
 
 if [ "$(uname -s)" = "Darwin" ]; then
   GROUP_ID=100
+  if (dscl . -read /Groups/docker); then 
+      DOCKER_GROUP_ID=$(dscl . -read /Groups/docker| awk '($1 == "PrimaryGroupID:") { print $2 }')
+    else
+      # if Docker post-install steps to manage as non-root user not performed - will use dummy gid
+      # see https://docs.docker.com/engine/install/linux-postinstall/ for more info
```

Comment:
Isn't it better to echo this information?

Reply:
Good point - echo that we recommend completing post-install steps and configuring to manage Docker as a non-root user

Reply:
Fixed, and new PR directly to the beam was provided https://github.com/apache/beam/pull/13545 so this one will be closed.